src/io/register_loader_saver_minimizer.cpp

```diff
@@ -36,6 +36,27 @@ void register_loader_saver_minimizer() {
     });
 }
 
+void register_loader_saver_minimizer_paths() {
+    std::uint32_t magic_number = gbwtgraph::MinimizerHeader::TAG;
+    std::string magic_string(reinterpret_cast<char*>(&magic_number), sizeof(magic_number));
+
+    Registry::register_bare_loader_saver_with_magic<gbwtgraph::MinimizerIndexXL>("MinimizerIndexXL", magic_string, [](istream& input) -> void* {
+        gbwtgraph::MinimizerIndexXL* index = new gbwtgraph::MinimizerIndexXL();
+
+        // Load it. In case of a failure, this will:
+        // * Throw an exception if sanity checks fail.
```

Comment:
I think we probably want to call sanity checks something else, since it's considered somewhat rude to people who aren't, strictly speaking, sane.